lib/model/SwitchingChannel.js

```diff
@@ -33,6 +33,32 @@ module.exports = class SwitchingChannel {
     return this._cache.triggerCapabilities;
   }
 
+  // { 'switch to channel key': array of (merged) ranges, ... }
+  get triggerRanges() {
+    if (!('triggerRanges' in this._cache)) {
+      let ranges = {};
+
+      for (const cap of this.triggerCapabilities) {
+        if (!(cap.switchTo in ranges)) {
+          ranges[cap.switchTo] = [cap.range];
+        }
+        else {
+          let chRanges = ranges[cap.switchTo];
```

Comment:
Maybe we should also create a new class for range since `[1]` for range end is not really intuitive